api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -223,12 +301,18 @@ public static JavaUnit parseUnit(final String data)
          if (unit != null)
             return unit;
       }
-      throw new ParserException("Cannot find JavaParser capable of parsing the requested data");
```

Comment:
This is still required for cases when roaster-jdt is not in classpath

Reply:
Valid point. Will change this